ChangeLog

```diff
@@ -1,5 +1,12 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.x branch released xxxx-xx-xx
+
+Bugfix
+   * Fix compilation error on Mingw32 when `_TRUNCATE` is defined. Use `_TRUNCATE`
+     only if `__MINGW32__` not defined. Fix suggested by Thomas Glanzmann and
```

Comment:
Please note that ` (backticks) are markdown and the ChangeLog is not written in that format.